modules/search/search_block_meta.go

```diff
@@ -0,0 +1,51 @@
+package search
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+
+	"github.com/google/uuid"
+	tempo_io "github.com/grafana/tempo/pkg/io"
+	"github.com/grafana/tempo/tempodb/backend"
+)
+
+type SearchBlockMeta struct {
```

Comment:
i agree with the linter on this one. `search.BlockMeta` would be better.

it does feel like this and the traditional blocks are diverging. We need some internal discussion about what the long term vision is here. I don't mind there being a separate set of "search blocks" if that's what makes sense, but we just need to discuss what data is stored reduce storage costs.